ci-operator/config/openshift-kni/cnf-features-deploy/openshift-kni-cnf-features-deploy-master.yaml

```diff
@@ -64,21 +64,21 @@ tests:
   commands: make ci-job
   container:
     from: src
-- as: e2e-gcp-ovn
+- as: e2e-aws-ci-tests
   skip_if_only_changed: ^ztp
   steps:
-    cluster_profile: gcp
+    cluster_profile: aws-telco
     env:
-      COMPUTE_NODE_TYPE: n2-standard-8
+      BASE_DOMAIN: sysdeseng.devcluster.openshift.com
     test:
-    - as: e2e-gcp-ovn
+    - as: e2e-aws-ci-tests
```

Comment:
y not be explicit about ovn?
ie - e2e-aws-ovn-ci-tests?
I guess because all Telco care for is ovn..